proposals/lip-0053.md

```diff
@@ -279,7 +279,7 @@ def verifyPartnerChainOutboxRoot(ccu: CCU) -> None:
 
     # Verification for non-empty certificates.
     if len(ccu.params.certificate) > 0:
-        outboxKey = MODULE_NAME_INTEROPERABILITY + SUBSTORE_PREFIX_OUTBOX_ROOT + sha256(partnerchainID)
+        outboxKey = STORE_PREFIX_INTEROPERABILITY + SUBSTORE_PREFIX_OUTBOX_ROOT + sha256(OWN_CHAIN_ID)
         proof = {
             siblingHashes: outboxRootWitness.siblingHashes,
             queries: [{
```

Comment:
aren't we missing to fix the value in line 287 as state in [this comment](https://github.com/LiskHQ/lips/issues/291#issuecomment-1451664747)?